docs/docs/05-reusable-components.md

```diff
@@ -296,6 +296,21 @@ ReactDOM.render(<HelloMessage name="Sebastian" />, mountNode);
 This simplified component API is intended for components that are pure functions of their props. These components must not retain internal state, do not have backing instances, and do not have the component lifecycle methods. They are pure functional transforms of their input, with zero boilerplate.
 However, you may still specify `.propTypes` and `.defaultProps` by setting them as properties on the function, just as you would set them on an ES6 class.
 
+```javascript
+const HelloMessage = (props) => <div style={props.style} className={props.className}>Hello, {props.name}</div>;
+HelloMessage.propTypes = {
+  name: React.PropTypes.string,
+  className: React.PropTypes.string,
+  style: React.PropTypes.object
```

Comment:
Let's drop `style` & `className` from this completely - we can use `name` as the only prop and keep `HelloMessage` identical & short. We just would remove the passing of the prop at the render call.